test-acceptance/src/org/omegat/gui/align/AlignerWindowTest.java

```diff
@@ -144,7 +130,24 @@ public void testAligner() throws Exception {
         aligner.menuItem("align_menu_file").click();
         aligner.menuItem("align_menu_close_item").click();
         //
-        closeProject();
      }
 
+    @Override
+    protected void onSetUp() throws Exception {
+        super.onSetUp();
+        tmpDir = Files.createTempDirectory("omegat-sample-project-").toFile();
+        File projSrc = new File(PROJECT_PATH);
+        FileUtils.copyDirectory(projSrc, tmpDir);
+        FileUtils.forceDeleteOnExit(tmpDir);
+        JFrame pickerFrame = GuiActionRunner.execute(() -> {
+            AlignFilePickerController picker = new AlignFilePickerController();
+            picker.setSourceDefaultDir(tmpDir.toPath().resolve("source").toString());
+            picker.setDefaultSaveDir(tmpDir.toPath().resolve("tm").toString());
+            picker.setSourceLanguage(new Language("en"));
+            picker.setTargetLanguage(new Language("fr"));
+            return picker.show(null);
```

Comment:
It is not obvious `picker.show(null)` not set the frame visible. We should add method `init` or similar.